src/labb4_ETSN20/SearchInFile.java

```diff
@@ -10,23 +10,24 @@ public static void main(String args[]) throws FileNotFoundException{
 		
 		SearchInFile search = new SearchInFile();
 		search.searchLines(args[0], args[1]);
-		
+
 	}
-	
+
 	public void searchLines(String pattern, String fileName) throws FileNotFoundException{
 
-		
+
 		//läs filen
 		File file = new File ("fileName");
 		Scanner scan = new Scanner(file); //läs från filen(om filen inte finns, throw exception)
-		
-		while (scan.hasNextLine()) { 
-			String line = scan.nextLine();					
+
+		while (scan.hasNextLine()) {
+			String line = scan.nextLine();
 			if(line.contains(pattern)) {
 				System.out.println("Pattern found at line: " + line);
 			}
 		}
-		
+		//adding comment
+
 		scan.close();
 	}
 
```

Comment:
Overall simple to follow. Not more complicated than it has to be. I think the code fulfils the functionality.